doc/reference/networking/can_iso_tp.rst

```diff
@@ -0,0 +1,49 @@
+.. _can_iso_tp:
+
+ISO-TP Transport Protocol
+#########################
+
+.. contents::
+    :local:
+    :depth: 2
+
+Overview
+********
+
+ISO-TP is a transport protocol defined in the ISO-Standard ISO15765-2 Road
+vehicles - Diagnostic communication over Controller Area Network (DoCAN).
+Part2: Transport protocol and network layer services. As its name already
+implies, it is originally designed, and still used in road vehicle diagnostic
+over Controller Area Networks.
+Nevertheless, it's not limited to applications in
```

Comment:
For a webpage, the line length does not matter much but when reading this readme txt file from console, it would be more readable if the chapters have around equal length lines.

Reply:
Should look better now. :-)